src/lib/pipeline_configuration.py

```diff
@@ -63,9 +67,10 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
 
 
 class PipelineConfiguration(object):
-    DEV_MODE = False
+    DEV_MODE = True
     
-    PROJECT_START_DATE = isoparse("2019-04-12T00:00:00+03:00")
+    #TODO change start date to ("2019-04-19T00:00:00+03:00") before production time
+    PROJECT_START_DATE = isoparse("2019-03-12T00:00:00+03:00")
     PROJECT_END_DATE = isoparse("2019-05-28T09:00:00+03:00")
```

Comment:
Is the end date for the radio shows or for the follow-up surveys?

Reply:
End of follow up surveys. This is however provisional as it may change as the project progresses.